code/modules/projectiles/projectile/energy/nuclear_particle.dm

```diff
@@ -27,7 +27,7 @@
 
 /obj/projectile/energy/nuclear_particle/on_hit(atom/target, blocked, pierce_hit)
 	if (ishuman(target))
-		radiation_pulse(target, max_range = 0, threshold = RAD_FULL_INSULATION)
+		radiation_pulse(target, max_range = 0, threshold = RAD_FULL_INSULATION, chance = 100)
```

Comment:
Why not `SSradiation.irradiate(target)`?

Reply:
Not sure why it wasn't like that originally, but no reason not to I guess.

Reply:
Oh there's an ishuman check in the projectile's proc apparently.